Classes/Fusion/Helper/NodeInfoHelper.php

```diff
@@ -103,6 +103,7 @@ public function renderNodeWithMinimalPropertiesAndChildrenInformation(
             // ensure _isHidden is sent to hidden nodes are correctly shown in the tree.
             // TODO: we should export this correctly named, but that needs changes throughout the JS code as well.
             '_hidden' => $node->tags->withoutInherited()->contain(SubtreeTag::disabled()),
+            '_hiddenByAncestors' => $node->tags->onlyInherited()->contain(SubtreeTag::disabled()),
             '_hiddenInIndex' => $node->getProperty('hiddenInMenu'),
             '_hasTimeableNodeVisibility' =>
```

Comment:
all this stuff is slightly getting more and more bloated ...  i dont like that :D maybe we should do something similar to what we have done with other data: use bit masks ... this is especially useful to denote that there is likely of all the things only one state that we really want to highlight in the ui.

https://github.com/neos/neos-ui/blob/81d01bed49549acb67c1637fdee99b38140af501/packages/neos-ui-redux-store/src/CR/Workspaces/index.ts#L12-L17

aside from this the chnage looks good :D Code wise it looks fine and should work ... is there anything missing than to disucss if we like to keep introducing data bloat :D ?